src/pydata_sphinx_theme/assets/styles/extensions/_pydata.scss

```diff
@@ -6,3 +6,9 @@
 .xr-wrap[hidden] {
   display: block !important;
 }
+
+// ipywidgets
+.jp-OutputArea-output.lm-Widget {
+  // override overflow:hidden rule from Lumino (.lm-Widget) to allow scrolling
+  overflow: auto;
```

Comment:
Note: I tested that this works for both nbsphinx and myst-nb (works = allows horizontal scrolling for wide table inside an ipywidget).

I'm wondering if there's some way to test this automatically, or if we could build two example pages from the same identical notebook, one through myst-nb and the other through nbsphinx.